README.rst

```diff
@@ -34,43 +34,66 @@ You can use ``pip`` to install codespell with e.g.:
 Usage
 -----
 
-For more in depth info please check usage with ``codespell -h``.
+Below are some simple usage examples to demonstrate how the tool works.
+For exhaustive usage information, please check the output of ``codespell -h``.
+
+Run codespell in all files of the current directory:
+
+.. code-block:: sh
+
+    codespell
+
+Run codespell in specific files or directories (specified via their names or glob patterns):
+
+.. code-block:: sh
+
+    codespell some_file some_dir/ *.ext
 
 Some noteworthy flags:
 
 .. code-block:: sh
 
     codespell -w, --write-changes
 
-The ``-w`` flag will actually implement the changes recommended by codespell. Not running with ``-w`` flag is the same as with doing a dry run. It is recommended to run this with the ``-i`` or ``--interactive`` flag.
+..
+
+    The ``-w`` flag will actually implement the changes recommended by codespell. Not running with ``-w`` flag is the same as with doing a dry run. It is recommended to run this with the ``-i`` or ``--interactive`` flag.
 
 .. code-block:: sh
 
     codespell -I FILE, --ignore-words=FILE
 
-The ``-I`` flag can be used for a list of certain words to allow that are in the codespell dictionaries. The format of the file is one word per line. Invoke using: ``codespell -I path/to/file.txt`` to execute codespell referencing said list of allowed words. **Important note:** The list passed to ``-I`` is case-sensitive based on how it is listed in the codespell dictionaries.
+..
+
+    The ``-I`` flag can be used for a list of certain words to allow that are in the codespell dictionaries. The format of the file is one word per line. Invoke using: ``codespell -I path/to/file.txt`` to execute codespell referencing said list of allowed words. **Important note:** The list passed to ``-I`` is case-sensitive based on how it is listed in the codespell dictionaries.
```

Comment:
These new added indentations will make it render incorrectly. You can "view file" to see how these are rendered as block quotes now -- clicking it:

<img width="356" alt="Screenshot 2022-12-12 at 9 24 05 AM" src="https://user-images.githubusercontent.com/2365790/207069644-ebe062fb-f826-4de8-9549-ad0dc27e9c26.png">

Will show the problem:

<img width="384" alt="Screenshot 2022-12-12 at 9 24 35 AM" src="https://user-images.githubusercontent.com/2365790/207069726-16709236-7343-44d4-b28c-db1408277456.png">